app/components/file_component.rb

```diff
@@ -7,7 +7,16 @@ def initialize(viewer:)
 
   attr_reader :viewer
 
+  delegate :purl_object, to: :viewer
+  delegate :citation_only?, :embargoed?, to: :purl_object
+
   def message
     viewer.authorization.message
   end
+
+  # If this method is false, then display the "content not available" banner
+  # We exclude the embargoed state, because we prefer to show the embargo banner and we don't want two banners
+  def display_not_available_banner?
+    citation_only? && !embargoed?
```

Comment:
I think this needs to be in the auth messages component. If there is something that is view stanford, special collections, etc and download none this banner is going to show up.

Reply:
It makes sense to wait until  https://github.com/sul-dlss/sul-embed/pull/2756 is merged then.

Reply:
@jcoyne #2756 is merged. Please rebase.